index.html

```diff
@@ -5,9 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <title>My Portfolio</title>
 </head>
 <body>
-    
+    <div class="header">
+        <a href="#"><img src="/image/images.png" width="100px"></a>
+        <nav> 
+        <a href="#" class="fa fa-bars"></a>
+        </nav>
+    </div>
+
+    <section>
+        <h1>I'm Abhishek. <br> Glad to see <br> you !</h1>
+        <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. </p>
```

Comment:
Your design
![image](https://github.com/Nomarg/My-Portfolio/assets/78251927/c0fabd1f-b42d-47a5-ac72-ba97c66bc1e9)

Figma
![image](https://github.com/Nomarg/My-Portfolio/assets/78251927/c03aabf3-9351-4d3c-afb8-7a9f74f0123d)


Kindly ensure to use the _(font, colors, images, text, margins)_ as defined for the template.

* If it's pretty and works it will be a wonderful website. 💪🤓